components/ui-api-layer/internal/domain/servicecatalog/instance_resolver_test.go

```diff
@@ -412,10 +428,12 @@ func TestInstanceResolver_ServiceInstanceBindableField(t *testing.T) {
 		expectedResult    bool
 		shouldReturnError bool
 	}{
-		{planErr: nil, classErr: nil, instanceBindable: true, expectedResult: true, shouldReturnError: false},
-		{planErr: nil, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: false},
-		{planErr: testErr, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: true},
-		{planErr: nil, classErr: testErr, instanceBindable: false, expectedResult: false, shouldReturnError: true},
+		{className: "className", planName: "planName", planErr: nil, classErr: nil, instanceBindable: true, expectedResult: true, shouldReturnError: false},
+		{className: "className", planName: "planName", planErr: nil, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: false},
+		{className: "className", planName: "planName", planErr: testErr, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: true},
+		{className: "className", planName: "planName", planErr: nil, classErr: testErr, instanceBindable: false, expectedResult: false, shouldReturnError: true},
+		{className: "className", planName: "", planErr: nil, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: false},
+		{className: "", planName: "planName", planErr: nil, classErr: nil, instanceBindable: false, expectedResult: false, shouldReturnError: false},
 	} {
 		className := "className"
 		planName := "planName"
```

Comment:
`className` and `planName` are no longer needed as you provide them in test data